com.github.bancika.diy_layout_creator.json

```
@@ -0,0 +1,133 @@
+{
+    "id": "com.github.bancika.diy_layout_creator",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "diylc",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/app/bin:/usr/bin",
+        "--env=JAVA_HOME=/app/jre",
+        "--env=JAVA=/app/jre/bin/java",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
```

…and here I disagree. You can't possibly know where users are going to store their data with the app like this.
```suggestion
        "--filesystem=home",
```

…and here I disagree. You can't possibly know where users are going to store their data with the app like this.